apps/web/src/pages/api/feedback/enabled.ts

```diff
@@ -3,5 +3,5 @@ import type { NextApiRequest, NextApiResponse } from "next";
 import { env } from "~/env.mjs";
 
 export default async function (req: NextApiRequest, res: NextApiResponse) {
-  return res.status(200).json({ enabled: !!env.FEEDBACK_WEBHOOK_URL });
+  return res.status(200).json(Boolean(env.FEEDBACK_WEBHOOK_URL));
```

Comment:
Can we revert this change?

I'd like to keep consistency with the rest of the API and always return a JSON

Reply:
Done [70dc1c9](https://github.com/Blobscan/blobscan/pull/602/commits/70dc1c99e6afb726f579ffb28de02bc810a096db)